examples/MonteCarloExamples/scenarioAnalyzeMonteCarlo.py

```diff
@@ -102,26 +102,29 @@
 
 import inspect
 import os
-FOUND_DATESHADER = True
+import holoviews as hv
+
+from Basilisk.utilities.MonteCarlo.AnalysisBaseClass import mcAnalysisBaseClass
+from bokeh.palettes import RdYlBu9
+
 try:
-    from Basilisk.utilities.datashader_utilities import DS_Plot, curve_per_df_component, pull_and_format_df
-    from Basilisk.utilities.MonteCarlo.AnalysisBaseClass import mcAnalysisBaseClass
-    from bokeh.palettes import Blues9, Reds9, Greens9, \
```

Comment:
I removed all the unused palettes but since this is an example file they should stay?

Reply:
No. Examples that demonstrate bad practice are bad examples.